### PR TITLE
luminous: qa/workunits/rbd: stress test `rbd mirror pool status --verbose`

### PR DESCRIPTION
**Pull request by trociny**
Fixes: https://tracker.ceph.com/issues/40923
Signed-off-by: Mykola Golub <mgolub@suse.com>
(cherry picked from commit 0866a2456b773843537681c768d93d1c9e7718a4)

Conflicts:
	qa/workunits/rbd/rbd_mirror_helpers.sh (trivial resolution)

Backport ticket: https://tracker.ceph.com/issues/41421

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`

</details>

### CODE REVIEW

**Comment by dillaman** [comment]
retest this please

**Comment by yuriw** [comment]
[wip-yuri5-testing-2019-10-18-1244-luminous](https://trello.com/c/mnR2A6fc/725-wip-yuri5-testing-2019-10-18-1244-luminous)

File: qa/workunits/rbd/rbd_mirror_helpers.sh [hunk 1]
```diff
@@ -544,11 +544,44 @@ test_status_in_pool_dir()
...
+    local state_pattern="$4"
+    local prev_last_update="$5"
+
+    local status_log=${TEMPDIR}/${cluster}-${pool}.mirror_status
+
+    rbd --cluster ${cluster} mirror pool status ${pool} --verbose --format xml \
+        > ${status_log}
+
+    local last_update state
+    last_update=$($XMLSTARLET sel -t -v \
```
Comment (dillaman) [inline qa/workunits/rbd/rbd_mirror_helpers.sh]: `$XMLSTARLET` is not defined in luminous

**Comment by trociny** [comment]
The results of running rbd suite with 'mirror' filter on luminous ceph branch and wip-41421-luminous suite branch:

http://pulpito.ceph.com/trociny-2019-10-21_08:27:36-rbd-luminous-distro-basic-smithi/

**Pull request merged**